definitions/lookups/lookup_survey_wave_questions.sqlx

```diff
@@ -0,0 +1,42 @@
+config {
+    type: "table",
+    description: "A table containing survey questions. Ensure any questions added here are also added to the UNPIVOT statement in question_responses",
+    columns: {
+      survey_wave_id: "The unique identifier for a survey. Get this from survey_waves table.",
+      question_id: "The unique identifier for a question. Get this from the questions table."
+    },
+    assertions : {
+      uniqueKey: ["survey_wave_id", "question_id"],
+    }
+}
+
+SELECT
+  *
+FROM
+  UNNEST( ARRAY<STRUCT<survey_wave_id STRING, question_id STRING>>[
+    -- wave 13 questions
+    -- ql1a
+    ('99bf804dece246925945905b20b33854', '1a60a6969bff49b683d401c8f3af093e'),
```

Comment:
@oliverroberts1-gds I'm leaning towards just using the business key (e.g. unhashed values) in the lookups to make it easier for analysts to update. What do you think?